dash/backend/src/modules/falco/controllers/falco.controller.ts

```diff
@@ -199,48 +186,57 @@ export class FalcoController {
         return result;
     }
 
-    @Post(':clusterId/rules')
+    @Post('/rules')
     @AllowedAuthorityLevels( Authority.SUPER_ADMIN, Authority.ADMIN )
     @UseGuards(AuthGuard, AuthorityGuard)
     async createFalcoRule(
       @Param('clusterId') clusterId: number,
-      @Body() rule: FalcoRuleDto
+      @Body() rule: FalcoRuleCreateDto
     ): Promise<FalcoRuleDto> {
-        if (rule?.clusterId !== clusterId) {
-            throw new HttpException({ message: 'Cluster id in path does not match cluster ID in body.' }, HttpStatus.BAD_REQUEST);
-        }
         delete rule.id;
         return await this.falcoService.createFalcoRule(rule);
     }
 
-    @Get(':clusterId/rules')    @AllowedAuthorityLevels( Authority.SUPER_ADMIN, Authority.ADMIN, Authority.READ_ONLY )
+    @Get('/rules')    @AllowedAuthorityLevels( Authority.SUPER_ADMIN, Authority.ADMIN, Authority.READ_ONLY )
     @UseGuards(AuthGuard, AuthorityGuard)
-    async listActiveFalcoRulesForCluster(@Param('clusterId') clusterId: number): Promise<FalcoRuleDto[]> {
-        return this.falcoService.listActiveFalcoRulesForCluster(clusterId);
-
+    async listActiveFalcoRulesForCluster(): Promise<FalcoRuleDto[]> {
```

Comment:
Are you filtering on the frontend now?

Reply:
There is currently no filtering, as there is now only the list of all rules